patroni/ctl.py

```diff
@@ -1602,18 +1618,24 @@ def output_members(cluster: Cluster, name: str, extended: bool = False,
         for member in sort(c['members']):
             logging.debug(member)
 
-            lag = member.get('lag', '')
-
             def format_diff(param: str, values: Dict[str, str], hide_long: bool):
                 full_diff = param + ': ' + values['old_value'] + '->' + values['new_value']
                 return full_diff if not hide_long or len(full_diff) <= 50 else param + ': [hidden - too long]'
             restart_reason = '\n'.join([format_diff(k, v, fmt in ('pretty', 'topology'))
                                         for k, v in member.get('pending_restart_reason', {}).items()]) or ''
 
+            receive_lag, replay_lag = member.get('received_lag', ''), member.get('replayed_lag', '')
+            receive_lsn, replay_lsn = member.get('received_lsn', ''), member.get('replayed_lsn', '')
+            receive_lag = round(receive_lag / 1024 / 1024) if isinstance(receive_lag, int) \
+                else '' if receive_lag == 'unknown' else receive_lag
+            replay_lag = round(replay_lag / 1024 / 1024) if isinstance(replay_lag, int) \
+                else '' if replay_lag == 'unknown' else replay_lag
```

Comment:
What about backward compatibility, when new patronictl is used with old Patroni?
I would suggest to pick values of `lsn` and `lag` and show them either in Receive or Replay columns depending on whether State is `streaming` or not.